lib/lago/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Lago
-  VERSION = '0.2.0'
+  VERSION = '0.3.0'
 end
```

Comment:
I guess that we should change version just before new release since there are multiple changes at the time that will end up in the new release. Do you agree?

Reply:
:+1: I agree. Plus, maybe we should align the library version to the API one : `1.X.Y`, what do you think?